frontend/Dockerfile

```diff
@@ -1,14 +0,0 @@
-FROM node:16.18.0
```

Comment:
This file needed for run project on computers of other developers.
Also Dockerfile used for deployment to production server.

Containerize it back, please.